lib/EnvironmentPlugin.js

```diff
@@ -38,7 +38,7 @@ class EnvironmentPlugin {
 				});
 			}
 
-			defs[`process.env.${key}`] = value === undefined ? "undefined" : JSON.stringify(value);
+			defs[`process.env.${key}`] = typeof value === undefined ? "undefined" : JSON.stringify(value);
```

Comment:
Should be changed to `typeof value === undefined` if to work properly.
Why change it from `value === undefined`?